src/foam/comics/v2/DAOControllerConfig.js

```diff
@@ -119,27 +119,16 @@ foam.CLASS({
       }
     },
     {
-      class: 'Boolean',
-      name: 'createEnabled',
-      documentation: 'If set to false, the "Create" button will not be visible.',
-      value: true
+      class: 'foam.mlang.predicate.PredicateProperty',
+      name: 'create'
```

Comment:
Doesn't this cause a name conflict?

Reply:
@olhabn revert to create/edit/deletedEnabled. 
and add default Predicate of TRUE.

Reply:
fixed https://github.com/foam-framework/foam2/pull/3234